src/adapter/cloudflare-workers/serve-static.ts

```diff
@@ -46,7 +47,7 @@ export const serveStatic = <E extends Env = Env>(
     })
 
     if (content) {
-      const mimeType = getMimeType(path)
+      const mimeType = getMimeType(path, options.mime)
```

Comment:
I would suggest using the default mime types if there were no mime types specified.

```ts
let mimeType: string | undefined = undefined
if (options.mime) {
  mimeType = getMimeType(path, options.mime) ?? getMimeType(path)
} else {
  mimeType = getMimeType(path)
}
```